pytorch_lightning/trainer/connectors/logger_connector/result.py

```diff
@@ -604,6 +605,16 @@ def cpu(self) -> "ResultCollection":
         """Move all data to CPU."""
         return self.to(device="cpu")
 
+    def sync(self) -> None:
+        for result_metric in self.result_metrics:
+            if result_metric.is_tensor:
+                result_metric.sync()
+
+    def unsync(self) -> None:
+        for result_metric in self.result_metrics:
+            if result_metric.is_tensor and result_metric._is_synced:
```

Comment:
I think we need to update

```
torchmetrics>=0.4.0
```

to

```
torchmetrics>=0.4.1
```

in `requirements.txt` to access `_is_synced`


```python
AttributeError: 'DummyMeanMetric' object has no attribute '_is_synced'
```

Reply:
Done !